src/SearchResults.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import Moment from "moment";
+
+function SearchResults(props) {
+  console.log(props);
+
+  return (
+    <div class="table-responsive">
+      <table class="table table-bordered table align-top">
+        <thead class="table-dark">
+          <tr>
+            <th scope="col">id</th>
+            <th scope="col">Title</th>
+            <th scope="col">First Name</th>
+            <th scope="col">SurName</th>
+            <th scope="col">Email</th>
+            <th scope="col">Room ID</th>
+            <th scope="col">check in date</th>
+            <th scope="col">check out date</th>
+            <th scope="col"> Number of Nights </th>
```

Comment:
you could even have a think about a dynamic way to get the table headings here, so you don't have to keep updating it if the data your received changed (for example imagine if more columns were added, or some were removed). You get each of these keys with each object, so have a think how you could grab the keys without having to type them all out